### PR TITLE
bot: small changes for email and result

### PR DESCRIPTION
**Pull request by tedd-an**
Small changes to add result.txt and attach result files to email.

Fixes: #298, #299, #300

### CODE REVIEW

File: bot/config.py.sample [hunk 1]
```diff
@@ -104,4 +104,14 @@ z['scheduler'] = {
     'friday': '20:00',
 }
 
+# ****************************************************************************
+# Post run configuration
+#
+# To do any post run tasks. It runs 'script' and followed by 'cleanup'
+# ****************************************************************************
+z['postrun'] = {
+    'script': '/path/to/script',
```
Comment (MariuszSkamra) [inline bot/config.py.sample]: I see you are running 'script' right after 'cleanup' so I guess you could simply provide a list of paths to scripts and just run all of those in for loop.
z['postrun'] = ['/path/to/script', '/path/to/cleanup/script']

This way you will not duplicate the code [1]

File: bot/zephyr.py [hunk 2]
```diff
@@ -443,6 +444,29 @@ def main(cfg):
...
+        if 'script' in postrun:
+            script = postrun['script']
+            if os.path.isfile(script):
+                script_proc = subprocess.Popen(shlex.split(script),
+                                               shell=True)
+                if script_proc.wait():
+                    print("Failed to run post script")
+
+        # Run post run script
+        if 'cleanup' in postrun:
```
Comment (MariuszSkamra) [inline bot/zephyr.py]: [1] here

Comment (tedd-an) [reply bot/zephyr.py]: I was going to use this routine to upload the test result to Zephyr TestRail site and reset the VM. I need to more think about it...   :)

File: bot/config.py.sample (hunk 1 above)
Comment (tedd-an) [reply bot/config.py.sample]: Good idea.

**Pull request merged**